games-puzzle/tanglet/tanglet-1.6.1.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg
+
+DESCRIPTION="Single player word finding game based on Boggle"
+HOMEPAGE="https://gottcode.org/tanglet/"
+SRC_URI="https://gottcode.org/${PN}/${P}-src.tar.bz2"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```

Comment:
No need for empty IUSE, remove the line.

I know it's just a copy of the old ebuild, but most games@ ebuild aren't a good reference, nice to cleanup things like this on bumps (I've been trying to improve games@ ebuilds slowly but there's 777 packages and hadn't gotten to tanglet yet, help is appreciated especially if want these fixed earlier).